README.md

```diff
@@ -95,9 +98,21 @@ To use this mode, your SFCC instance must be configured to allow the origin of t
 }
 ```
 
+#### Allowing the extension in OCAPI settings
+
+To allow the extension to access product data in your SFCC environment, you will also need to add this to your SFCC OAuth client's `allowed_origins`:
+
+```json
+{
+  "allowed_origins": [
+    "https://product-selector.extensions.content.amplience.net"
+  ]
+}
+```
+
 ### SFCC (proxy)
 
-The extension works with 'list of text' properties and supports the following parameters:
+To use this mode, you will also need to ensure that "Allow same origin" is disabled within the [extension's settings](#Permissions). You will also need to build and host your own version of [sfcc-product-search-proxy](https://github.com/amplience/sfcc-product-search-proxy). The extension works with 'list of text' properties and supports the following parameters:
```

Comment:
Must this be disabled?

Reply:
Yeah it should be enabled
<img width="855" alt="Screenshot 2021-11-26 at 12 21 28" src="https://user-images.githubusercontent.com/468889/143580376-b80b1f5c-af5d-489a-9d90-e5af0020a3b4.png">
)

Reply:
This is for the proxied version of the extension, with the [proxy](https://github.com/amplience/sfcc-product-search-proxy) using a null origin. For this to work, the extension requires "Allow same origin" disabling.